packages/mobile/src/screens/update-required-screen/NewVersionPrompt.tsx

```diff
@@ -0,0 +1,65 @@
+import { View } from 'react-native'
+
+import IconDownload from 'app/assets/images/iconDownload.svg'
+import { Button, GradientText, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  contentContainer: {
+    paddingTop: spacing(32),
+    paddingBottom: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  header: {
+    fontSize: 24,
+    lineHeight: 52,
+    textAlign: 'center',
+    textShadowOffset: { height: 2, width: 0 },
+    textShadowRadius: 4,
+    textShadowColor: 'rgba(162,47,235,0.2)'
+  },
+  text: {
+    textAlign: 'center',
+    marginBottom: spacing(8),
+    marginTop: spacing(6)
+  }
+}))
+
+type NewVersionPromptProps = {
+  headerText: string
+  contentText: string
+  buttonText: string
+  /** Passing a `url` will make the CTA element a link */
+  url?: string
```

Comment:
If we wanted to be fancy it could be url or onPress required

Reply:
wait you can do this??

Reply:
yeah i think there are special types that can make this, but i think simplest is something like

Reply:
ohh makes sense! ty